code/player/Player.Jobs.cs

```diff
@@ -7,11 +9,140 @@ public partial class Player
     [BindComponent]
     public Jobs.PlayerJob Job { get; }
 
+    //The default cooldown time for setting the forgiveness timer
+    float cooldownTime => 30.0f;
```

Comment:
Naming: `cooldownTime` is too vague. This needs to be something like `JobCooldownDuration` or something